CMakeLists.txt

```diff
@@ -354,7 +354,14 @@ target_include_directories(${PROJECT_NAME} PRIVATE
 if (UNIX)
     target_compile_options(${PROJECT_NAME} PRIVATE -Wall -Wno-sign-compare -Wno-unused-function -Werror ${LLVM_CPP_FLAGS})
     # Security options
-    target_compile_options(${PROJECT_NAME} PRIVATE -fstack-protector-strong -fno-strict-float-cast-overflow
+    if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+      # Using the clang compiler
```

Comment:
The rest of the file has indentation by 4 spaces, please fix.

Reply:
Ah, my bad.
Fixed it now.
Do you want me to squash the commits?

Reply:
Don't worry about it, I'll squash it on the merge.